docker/pytorch/Dockerfile

```diff
@@ -8,6 +8,19 @@ ARG DEBIAN_FRONTEND=noninteractive
 # won't be able to build the docker image.
 RUN rm -f /usr/local/cuda-11.3/cuda-11.3
 
+# update repository keys
+# https://developer.nvidia.com/blog/updating-the-cuda-linux-gpg-repository-key/
+ARG LINUX_DISTRO
+RUN rm -f /etc/apt/sources.list.d/cuda.list
```

Comment:
Can these `RUNs` be combined into a singular command, and check to see if the `CUDA_VERSION` variable is defined? (See the check on line 52. The build error is happening as the CPU image does not have cuda.)

Reply:
Done